src/styles/main.scss

```diff
@@ -1,3 +1,22 @@
-body {
+@import "./utilts/variables";
+@import "./utilts/normilize";
+@import "./utilts/reset";
+@import "./utilts/extends";
+@import "./blocks/catalog.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/card.scss";
+@import "./blocks/opinion.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+}
+
+.body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
 }
```

Comment:
Part of this is duplicated in `normalize.scss`, and `main.scss` isn't a right place for this.